exercises/concept/ellens-alien-game/alien_class.py

```diff
@@ -0,0 +1,14 @@
+## Create the Alien class here
+
+
+def new_alien_list(positions):
+    """
+
+    :param positions: list - a list of tuples of (x, y) coordinates
+    :return: list - a list of alien objects
+    
+    Function that takes a list of positions and creates one alien
```

Comment:
I'm not sure which docstring style is being followed here. The numpy style has the summary/description before the param/return list.